python/lsst/qa/explorer/joinFeatures.py

```diff
@@ -0,0 +1,157 @@
+# This file is part of qa explorer
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Makes one large training set for the classifier.
+"""
+
+import pandas as pd
+
+import lsst.pex.config as pexConfig
+import lsst.daf.persistence.butlerExceptions
+from lsst.pipe.base import Task
+from .parquetTable import ParquetTable
+
+__all__ = ["JoinFeaturesTask", "JoinFeaturesConfig"]
+
+
+class JoinFeaturesConfig(pexConfig.Config):
+    """Config for the `JoinFeaturesTask`.
+
+    Notes
+    -----
+    Currently only the options of ``galaxies``, ``stars`` and ``all`` are supported for sourceTypes. These
+    allow only stars, only galaxies or all sources to be used from each rerun passed to the `JoinFeaturesTask`
+    they must be in the same order as the reruns.
+    """
+    sourceTypes = pexConfig.ListField(
+        doc="Type of sources used from each rerun.",
+        dtype=str,
+        default=["galaxies", "stars", "stars", "all"]
```

Comment:
This is because only the stars were used from two different input repos. I have added this to the notes for the JoinFeaturesConfig so hopefully it will be more obvious.